### PR TITLE
add renovate

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
You can see this in action at https://github.com/evgeni/foreman-infra/pulls :)

**Comment by evgeni** [comment]
Instead of using a github action, we could also use the Renovate app (https://github.com/marketplace/renovate) which would not have the token issue, I am happy with either, but I'd like the feature of $thing bumping our deps.

File: .github/workflows/renovate.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - cron: '4 3 * * *'
+
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v39.1.1
```
Comment (ekohl) [inline .github/workflows/renovate.yml:15]: This will generate a lot of noise. Not too happy with that.

**Review by ekohl — CHANGES_REQUESTED** [review]
Because of the noise in updating the GH action version I'm leaning to the app.

File: .github/workflows/renovate.yml (hunk 1 above)
Comment (evgeni) [reply .github/workflows/renovate.yml:15]: yepp, they don't have a "rolling" branch :(

**Comment by evgeni** [comment]
Starting setting up the app in https://github.com/theforeman/foreman-infra/pull/1961

**Comment by evgeni** [comment]
closing in favor of https://github.com/theforeman/foreman-infra/pull/1961

**Pull request closed without merging**